### DISCUSSION TITLE
Field alias not working for query models

### DISCUSSION THREAD

**Discussion by potapov-ev** [fastapi/fastapi#13109] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class Queries(BaseModel):
    pay_method: str = Field(default=None, alias="payMethod", validation_alias="payMethod") 
    bill_type: str = Field(default=None, alias="billType")


@app.get("/")
def read_root(queries: Queries = Query()):
    return queries
```

### Description

Hello)

Fields `alias` not working for `Query` model
Get None or default value

`validation_alias` working for me
but it seems like a bug

something similar happened here
https://github.com/fastapi/fastapi/discussions/10024

Thank you in advance

### Operating System

macOS

### Operating System Details

sonoma 14.5

### FastAPI Version

0.115.2

### Pydantic Version

2.10.3

### Python Version

3.11.10

**Comment by YuriiMotov** [comment, marked as answer]
It's a bug.
There is already issue: https://github.com/fastapi/fastapi/issues/12402
